model/DataStore/PersistDataService.php

```diff
@@ -99,25 +125,28 @@ private function persistArchive(string $deliveryId, array $params): void
      * @throws common_exception_Error
      * @throws common_exception_NotFound
      */
-    private function moveExportedZipTest(string $folder, string $deliveryId, array $params): void
+    private function moveExportedZipTest(string $folder, string $deliveryOrTestId, array $params): void
```

Comment:
nitpick: I would rename `deliveryOrTestId` to `entityId` in this context it does not look to be important to know that it can have this or that.

Reply:
Replaced to `resourceId` in this case